src/main/java/seedu/address/ui/JobCard.java

```diff
@@ -51,6 +51,6 @@ public JobCard(Job job, int displayedIndex) {
         job.getRequirements().stream()
                 .sorted(Comparator.comparing(requirement -> requirement.tagName))
                 .forEach(requirement -> requirements.getChildren().add(new Label(requirement.tagName)));
-        matches.setText("Employee count - " + job.getMatches().size());
+        matches.setText("Employee count - PLACEHOLDER");
```

Comment:
Does anyone know when is the various UI elements updated exactly? 

I'm worried about a possible running time inefficiency issue if I implement a search.

Reply:
Resolved! I just removed this entirely. Let me know if you guys want to keep it or not. Another comment below related to why I deleted.